re-former/app/views/users/new.html.erb

```diff
@@ -0,0 +1,50 @@
+<h1>New User</h1>
+
+<!--
+<form action="/users" accept-charset="UTF-8" method="post">
+<input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+
+  <label for="username">User Name:</label>
+  <input type="text" name="user[username]" id="username">
+  <br><br>
+  <label for="email">Email:</label>
+  <input type="email" name="user[email]" id="email">
+  <br><br>
+  <label for="password">Password:</label>
+  <input type="password" name="user[password]" id="password">
+  <br><br>
+  <button type="submit" name="submit">Save</button>
+</form>
+-->
+<!--
+<%= form_tag(model: @user) do %>
+  <%= label_tag(:username, "User Name:") %>
+  <%= text_field(:user,:username) %><br><br>
+
+  <%= label_tag(:email, "Email:") %>
+  <%= email_field(:user,:email) %><br><br>
+
+  <%= label_tag(:password, "Password:") %>
+  <%= password_field(:user,:password) %><br><br>
+
+  <%= submit_tag("Save") %>
+<% end %>
+-->
+
+<%= form_for( @user)  do |f| %>
```

Comment:
- [ ] **[OPTIONAL]** To DRY our views a little we can: 
  - move error messages section as it is repeated in `new.html.erb` and `edit.html.erb`into `application.html.erb`
  - Use partials to dry our forms since they have the same code. [partials](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials)